.buildkite/scripts/build_and_upload_images.sh

```diff
@@ -36,6 +36,8 @@ echo "--- Building all ${IMAGE_TAG} images"
 # all the buildx file introspection (and thus build-target awareness)
 # localised here.
 make build-image-prerequisites
+
+docker buildx bake --file="${BUILDX_BAKE_FILE}" --progress "plain" "${BUILDX_TARGET}"
 docker buildx bake --file="${BUILDX_BAKE_FILE}" --progress "plain" --push "${BUILDX_TARGET}"
```

Comment:
I'm not sure if this is going to actually do what you want. `--push` is just a shorthand for "the output of this build is a registry image at XXX registry".

True, the first `bake` invocation will just build the images locally, and the second one will essentially be a no-op due to everything being cached. I don't think that building is the issue, though.